build-mbl/run-me.sh

```diff
@@ -23,6 +24,38 @@ cleanup() {
     fi
 }
 
+build_script_for_project() {
+    project=${1:?}
+    case "$1" in
+        mbl)
+            printf "%s\n" "build.sh"
+            ;;
+        poky)
+            printf "%s\n" "build-poky.py"
+            ;;
+        *)
+            printf "Unrecognized project \"%s\"" "$project" >&2
+            exit 5
+            ;;
+    esac
+}
+
+dockerfile_for_project() {
+    project=${1:?}
+    case "$1" in
+        mbl)
+            printf "%s\n" "Dockerfile"
```

Comment:
doesn't make sense to create a directory for mbl as well?

Reply:
For consistency, yes. The intention was to leave all of the "mbl" files at the top level and put special test versions into folders.

Reply:
I guess we are going to a direction where we need to build different flavours of things, so everything is special :)